src/javaRosa.js

```diff
@@ -310,8 +310,9 @@ define([
         updateForMug: function (mug, defaultLabelValue) {
             // set default itext id/values
             if (!mug.options.isDataOnly) {
+                var item;
```

Comment:
What do you think of renaming this variable to `label` now that it's scope reaches across the function to the help itext item as well?